### ISSUE TITLE
Question about Setting of torque parameters

### ISSUE THREAD

**Issue by MWHYNOT** [accel-sim/accel-sim-framework#192]
Excuse me, when I use the command `./util/job_launching/run_simulations.py -B rodinia_2.0-ft -C RTX3070-SASS -T ./hw_run/traces/device-0/11.2/ -N rodinia2`,
 I will get the output like below:
“Running Simulations with GPGPU-Sim built from
accelsim-commit-2260456ea5e6a1420f5734f145a4b7d8ab1d4737_modified_0.0

Using configs: RTX3070-SASS
Benchmark: rodinia_2.0-ft
Run Subdir = RTX3070-SASS
Parameters =
#SASS
#SASS-Driven Accel-Sim

Base config file = /home/gpgpu/newdisk/wyk/accel-sim-framework/gpu-simulator/gpgpu-sim/configs/tested-cfgs/SM86_RTX3070/gpgpusim.config
Job 1000 queued (backprop-rodinia-2.0-ft-4096___data_result_4096_txt RTX3070-SASS)
Job 1001 queued (bfs-rodinia-2.0-ft-__data_graph4096_txt___data_graph4096_result_txt RTX3070-SASS)
Job 1002 queued (hotspot-rodinia-2.0-ft-30_6_40___data_result_30_6_40_txt RTX3070-SASS)
Job 1003 queued (heartwall-rodinia-2.0-ft-__data_test_avi_1___data_result_1_txt RTX3070-SASS)
Job 1004 queued (lud-rodinia-2.0-ft-_v__b__i___data_64_dat RTX3070-SASS)
Job 1005 queued (nw-rodinia-2.0-ft-128_10___data_result_128_10_txt RTX3070-SASS)
Job 1006 queued (nn-rodinia-2.0-ft-__data_filelist_4_3_30_90___data_filelist_4_3_30_90_result_txt RTX3070-SASS)
Job 1007 queued (pathfinder-rodinia-2.0-ft-1000_20_5___data_result_1000_20_5_txt RTX3070-SASS)
Job 1008 queued (srad_v2-rodinia-2.0-ft-__data_matrix128x128_txt_0_127_0_127__5_2___data_result_matrix128x128_1_150_1_100__5_2_txt RTX3070-SASS)
Job 1009 queued (streamcluster-rodinia-2.0-ft-3_6_16_1024_1024_100_none_output_txt_1___data_result_3_6_16_1024_1024_100_none_1_txt RTX3070-SASS) ”

But the file **"sim_log.rodinia2.txt"** doesn't have the data, I think it should be **the problem about the Setting of torque parameters**,and when I use the command `qmgr -c "p s"`, I will get below:
"Create queues and set their attributes.

Create and define queue batch

create queue batch
set queue batch queue_type = Execution
set queue batch resources_default.nodes = 1
set queue batch resources_default.walltime = 200:00:00
set queue batch enabled = True
set queue batch started = True

Create and define queue std

create queue std

Set server attributes.

set server scheduling = True
set server acl_hosts = chezai-Super-Server
set server managers = root@chezai-Super-Server
set server operators = root@chezai-Super-Server
set server default_queue = batch
set server log_events = 511
set server mail_from = adm
set server scheduler_iteration = 600
set server node_check_rate = 150
set server tcp_timeout = 6
set server mom_job_sync = True
set server keep_completed = 300
set server submit_hosts = chezai-Super-Server
set server allow_node_submit = True
set server next_job_number = 1010  "

So I wonder  your **setting of torque parameters**，or can you give me some advice, I'd appreciate it if you could help me.

**Comment by JRPan** [comment]
Do you know if the jobs are running correctly? You can try `./util/job_launching/job_status.py`? 
Our server switched to slurm, and it was set up by the IT guys. So sorry cannot help much there. 

But You can use the procman to spawn the jobs locally. Just add `-l local` when using `run_simulations.py`

**Comment by MWHYNOT** [comment]
Thanks for the reply, I tried `./util/job_launching/job_status.py`, it appeared "UNKNOWN", and then I tried to add ` -l local` when using `run_simulations.py`, but it still didn't work.
Although there are such mistakes, I can still get the output file by the command `./justrun` manually in the corresponding benchmark directory, but I can't use the Accel-Sim Correlator due to this mistake.
I think something maybe wrong when I configured experiment, I will try again.Thanks again!
